# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mediasource/ContainerParser.cpp
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 dom/media/mediasource/ContainerParser.cpp
--- a/dom/media/mediasource/ContainerParser.cpp	Fri Jan 27 19:06:16 2023 +0000
+++ b/dom/media/mediasource/ContainerParser.cpp	Fri Jan 27 21:11:42 2023 +0200
@@ -19,9 +19,9 @@
 #ifdef MOZ_FMP4
 #  include "AtomType.h"
 #  include "BufferReader.h"
-#  include "ByteStream.h"
+#  include "Index.h"
 #  include "MP4Interval.h"
-#  include "SampleIterator.h"
+#  include "ByteStream.h"
 #endif
 #include "SourceBufferResource.h"
 #include <algorithm>